app/services/notification_service.rb

```diff
@@ -0,0 +1,21 @@
+class NotificationService
+  NOTIFICATION_TYPES = {
+    Comment       => [Notifications::CommentOnPost, Notifications::AlsoCommented],
+    Like          => Notifications::Liked,
+    StatusMessage => Notifications::Mentioned,
+    Conversation  => Notifications::PrivateMessage,
+    Message       => Notifications::PrivateMessage,
+    Reshare       => Notifications::Reshared,
+    Contact       => Notifications::StartedSharing
+  }.freeze
```

Comment:
Always wrapping the values into an array and getting rid of the `[*...]` below is probably a bit cleaner here.